### ISSUE TITLE
Boot issue

### ISSUE THREAD

**Issue by MrFluffyOven** [boydaihungst/treble_build_derpfest#26]
Hello, I am using the Motorola g(53) 5g, and the gsi is not even booting, it comes up with the Motorola splash screen and reboots, and is in a boot-loop

**Comment by boydaihungst** [comment]
If it is not even showing bootanimation: an owl beside of the "derpfest" text, then you must do something wrong or missing some steps. Like: forgot to wipe userdata, flash vbmeta, unlock bootloader, etc. Just to be clear, does td and ponces's aosp boot on your device?

**Comment by MrFluffyOven** [comment]
Yes, derpfest is the only one so far that's not booted, I've flashed vbmeta, disabled verity, deleted pointless partitions, and flashed the gsi via fastbootd, I have been doing this a while, I'm open to hearing if there is something you think i missed

**Comment by boydaihungst** [comment]
Alright. If you can go to recovery then give some logs : `adb shell "logcat -b all" > log.txt`

https://docs.halium.org/en/latest/porting/debug-build/dmesg.html

**Comment by MrFluffyOven** [comment]
I'm currently not near my PC, but I'm going home tonight, so as soon as I'm home I'll pull the logs for you
